torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1,4 +1,7 @@
-# mypy: ignore-errors
+# mypy: allow-untyped-defs
+
+from __future__ import annotations
```

Comment:
Do we actually need __future__ annotations? It prevents mypy and ruff from finding undefined annotations I think so we should try to avoid it when possible.

Reply:
ruff had issues with parsing it, but I haven't tested this behavior extensively in the latest version of mypy.

Reply:
In any case, without lazy annotation parsing I have to do strings everywhere and I don't like strings lol